docs/docs/widgets/image.md

```diff
@@ -6,21 +6,21 @@ title: Image
 
 The **Image** widget is used to display images in your app.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 |  <div style={{ width:"100px"}}> Properties </div> |  <div style={{ width:"100px"}}> Description </div> | 
 |:----------- |:----------- |
 | URL | Enter the URL of the image to display it on the widget. |
```

Comment:
Here widget needs to be changed to component
Also, you have made a different change for this particular line in the LTS version, please check it and make the constant changes in both the docs.